src/pages/pages/integrations.md

```diff
@@ -0,0 +1,44 @@
+---
+templateKey: default-page
+seo:
+  description: "Integrations for Ironic Bare Metal"
+  title: Ironic Bare Metal Integrations
+  twitterUsername: "@openstack"
+  url: "https://ironicbaremetal.org/"
+title: Integrations
+subTitle: "Ironic project Integrations and Tools"
+---
+
+The Ironic project has a long history of being driven by Cloud and Infrastruture operators seeking to solve their problems. This has resulted in a diverse set of capabilities, integrations, and ways to leverage ironic.
+
+<br>
+
+## Community Projects with Integration
```

Comment:
I'd split between the tools that deploy ironic (bifrost, kolla, startlingx) and the tools that use ironic (tripleo, metal3, kayobe)

Reply:
I'm trying to keep the list as one single consolidated list, not by precise purpose. Trying to demonstrate there are integrations  which exist, and then provide a little text to scope it.  I originally started in three sections and it just felt weird and consolidated down to what was in the change.

Reply:
And by consolidated list, I'm focusing on perception impact.

Reply:
Yeah, but I think the difference between "How do I get Ironic" and "What else uses Ironic" is very significant for potential users. These are completely different questions asked for different reasons.

Reply:
So if we take that path, then we duplicate some entries on the lists. That seems like it is just going to make it more complex to mentally parse that there *are* integrations which exist.

Reply:
This is also true

Reply:
I guess the question becomes, where is the happy middle ground :\ I'll look at it after lunch and see if I can find something.